juniper_official/Interfaces/interface-in-errors-iagent.rule

```diff
@@ -0,0 +1,126 @@
+/*
+ * Detects interface in errors and notifies when anomalies are found.
+ */
+healthbot {
+    topic interface.statistics {
+        description "Monitors and notify interface statistics i.e. link state, flaps, neighbor state, input & output traffic and errors";
+        synopsis "Interface statistics analyzer";
+        rule check-in-errors-iagent {
+            keys name;
+            synopsis "Interface in-error analyzer";
+            description "Collects the interface input error (errors (all), drops,discards, timeouts and runts) periodically and notifies in case of anomalies";
+            sensor interface-stats {
+                synopsis "Interfaces netconf based iAgent sensor definition";
+                description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables&views to collect data from network device";
+                iAgent {
+                    file interface-details.yml;
+                    table intfaceStats;
+                    frequency 30s;
+                    args interface_name {
+                        arg-value "{{interface-name}}";
+                    }
+                }
+            }
+            field error-threshold {
+                constant {
+                    value "{{error-threshold-variable}}";
+                }
+                type integer;
+                description "In errors count increase threshold";
+            }
+            field name {
+                sensor interface-stats {
+                    path name;
+                }
+                type string;
+                description "Interfaces to be monitored";
+            }
```

Comment:
Create all integer fields in rule file to support plots

Reply:
added integer fields